src/main/java/com/dawn/jat/illuminati/cms/controller/CmsController.java

```diff
@@ -6,7 +6,7 @@
 
 @Controller
 public class CmsController {
-    @RequestMapping("/cms")
+    @RequestMapping(value={"", "/cms", "/cms/*"})
```

Comment:
can we use "/" instead of ""?

Reply:
other than this, this is look good to me, it depends on you 👍

Reply:
Updated to use "/" instead of ""